app/jobs/send_donation_email_job.rb

```diff
@@ -0,0 +1,31 @@
+
+class SendDonationEmailJob < ActiveJob::Base
+  queue_as :default
+
+  require "mandrill"
+
+  def perform(*args)
+    # Do something later
+    m = Mandrill::API.new
+      message = {
+        :from_name=> "Puppify",
+        :from_email=>"codefordenver@gmail.com",
+        :to=> "dviramontes@gmail.com",
```

Comment:
We probably don't want people's e-mail addresses in the repository. Could we look at moving sensitive information into` ENV['MySecret']`?